app/views/deep_links/mobile.html.erb

```diff
@@ -0,0 +1,48 @@
+<div id="mobile-deep-link-spinner" class="flex flex-col">
```

Comment:
We'll need to make this markup accessible and handle it as well in the JS that shows/hides the different sections of markup.

We'll need to use [aria-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions). This will allow a screen reader to announce if there is a failure. We can use it in conjunction with the [aria-hidden](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute) attribute (toggling between true/false).

I'm done for today, but just pinging @aitchiss, when she starts her day, to drop her thoughts here as well. 👋🏻

If you search the codeabse for `aria-live`, you'll see some examples of this in the wild.

Reply:
Thank you for the note and suggestions @nickytonline!

From what I understand at first glance, we'd need to add `aria-live="polite"` to both divs ([this one](https://github.com/forem/forem/blob/953df231a6cf76512f1a415aaf70dcb63c4c4516/app/views/deep_links/mobile.html.erb#L1) and [this one](https://github.com/forem/forem/blob/953df231a6cf76512f1a415aaf70dcb63c4c4516/app/views/deep_links/mobile.html.erb#L6)) so they're announced.

On top of that, toggle `aria-hidden="true/false"` in the JS [here](https://github.com/forem/forem/blob/953df231a6cf76512f1a415aaf70dcb63c4c4516/app/views/deep_links/mobile.html.erb#L46) so they're available/hidden to the assistive technology when they become available/not-available.

I can make these changes (please correct me if I'm wrong though) tomorrow and also include any other suggestions @aitchiss may have 🙂

Reply:
the `invisible` class you're using will hide an item from assistive technology too - so you won't need to use `aria-hidden` in addition to it 🙂 

I would suggest `aria-live="polite"` for the `Opening the mobile app...` div, but `role="alert"` for the other ('alert' is generally used for errors which feels appropriate in this case!)

Reply:
Cool, the `role="alert"` vs `aria-live="polite"` is great insight to keep in mind for the future. Thanks a lot @aitchiss!